templates/components/products/card.html

```diff
@@ -53,10 +53,13 @@
                             <a href="{{pre_order_add_to_cart_url}}" data-event-type="product-click" class="button button--small card-figcaption-button">{{lang 'products.pre_order'}}</a>
                         {{/if}}
                         {{#if add_to_cart_url }}
-                            <a href="{{add_to_cart_url}}" data-event-type="product-click" class="button button--small card-figcaption-button">{{lang 'products.add_to_cart'}}</a>
+                            <span class="addCart-wrapper">
+                                <a href="{{add_to_cart_url}}" data-event-type="product-click" class="button button--small card-figcaption-button addCart">{{lang 'products.add_to_cart'}}</a>
```

Comment:
I believe we should use data attribute instead of css class here

Reply:
data-event-type="product-click" as other class names are not specifically unique for "Add to Cart" Button. As I see it by using this data attribute we will add extra event listeners that won't be in use.

Reply:
I mean new data-attribute, not event-type

Reply:
done